### PR TITLE
Add Hind to Steering Council

### PR DESCRIPTION
**Pull request by jaimergp**
Dear @conda/steering-council,

The voting period for @hind-m's application has concluded:

image

The vote reached quorum (10/14) and has **passed** (6/7).

This PR adds Hind to the CSV. @hind-m, please review the changes to include your email address and pronouns. Thanks!

### CODE REVIEW

**Comment by wolfv** [comment]
Welcome!

**Comment by jaimergp** [comment]
We still need Hind's email and pronouns 😬

**Comment by ocefpaf** [comment]
Welcome aboard!

**Comment by jaimergp** [comment]
@jezdez or any other org admin, can we add @Hind-M to the team? I could add Tania because she was already in the org, but Hind needs to be invited to the org first.

File: steering.csv [hunk 1]
```diff
@@ -14,4 +14,5 @@ jaimergp,jaime.rogue@gmail.com,Jaime Rodríguez-Guerra,Quansight,he/him
 kkraus14,keith.j.kraus@gmail.com,Keith Kraus,no funding, he/him
 baszalmstra,zalmstra.bas@gmail.com,Bas Zalmstra,prefix.dev,he/him
 beckermr,becker.mr@gmail.com,Matthew R. Becker,no funding,he/him
+hind-m,,Hind Montassif,QuantStack,
```
Comment (jezdez) [inline steering.csv:17]: The email address should have been filled in, to add them to the mailing list

**Comment by jezdez** [comment]
Welcome @Hind-M!

**Comment by jezdez** [comment]
> @jezdez or any other org admin, can we add @Hind-M to the team? I could add Tania because she was already in the org, but Hind needs to be invited to the org first.

Done!

File: steering.csv (hunk 1 above)
Comment (Hind-M) [reply steering.csv:17]:
```suggestion
hind-m,hind.montassif@gmail.com,Hind Montassif,QuantStack,she/her
```

**Pull request merged**